components/brave_wallet/browser/keyring_controller.cc

```diff
@@ -650,14 +689,16 @@ size_t KeyringController::GetAccountMetasNumberForKeyring(
   return account_metas->DictSize();
 }
 
+// This member function should not assume that the wallet is unlocked!
```

Comment:
can we also update the test case `KeyringControllerUnitTest.AccountMetasForKeyring` to get account meta when locked?

Reply:
Fixed and rebased in 10bd683f95363d743b2fb72a09278f0b5c3bb92e